adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -959,8 +959,13 @@
                         }
 
                         var loginUrl = kc.createLoginUrl(options);
-                        var ref = window.open(loginUrl, '_blank', o);
-
+                        var ref;
+                        if(window.cordova && window.cordova.InAppBrowser){
+                           // Use inappbrowser for IOS
+                           ref = window.cordova.InAppBrowser.open(loginUrl, '_blank', o);
+                        } else{
+                           ref = window.open(loginUrl, '_blank', o);
+                        }
```

Comment:
This is only handling one out of 4 places where window.open is called. Please add a windowOpen method to the adapters, then update the rest of the code to call adapter.windowOpen instead of window.open directly.

Reply:
Great idea! Going to do that.